src/reanimated2/PropsRegistry.ts

```diff
@@ -21,7 +21,7 @@ function flush() {
   viewTags = [];
 }
 
-function removeFromPropsRegistryOnUI(viewTags: number[]) {
+function removeFromPropsRegistryOnUI(innerViewTags: number[]) {
```

Comment:
Let's restore the original name of the argument here (`viewTags`) and rename the variable in line 7:
```diff
-let viewTags: number[] = [];
+const VIEW_TAGS: number[] = [];
```
```diff
-viewTags = [];
+VIEW_TAGS.length = 0; // clear the array
```